liota/dccs/iotcc.py

```diff
@@ -287,6 +302,56 @@ def prettify(self, elem):
         reparsed = minidom.parseString(rough_string)
         return reparsed.toprettyxml(indent="    ")
 
+    def _iotcc_json_structure(self):
```

Comment:
It might be better to use "actions" as method names, like "_get_iotcc_json_structure"